### PR TITLE
Up to WorldGuard 7.0, MC 1.13.1

### PR DESCRIPTION
**Pull request by FrostDelta123**
Up to WorldGuard 7.0, MC 1.13.1

### CODE REVIEW

File: src/wgextender/features/flags/FlagRegistration.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 public class FlagRegistration {
 
-	@SuppressWarnings("unchecked")
+	//Вот это надо убрать, к чему тут рефлексия? Флаги проще регистрируются
```
Comment (Shevchik) [inline src/wgextender/features/flags/FlagRegistration.java]: The reason why reflection exists here is to transform the old instance flags to support reloading of WGExtender

Comment (FrostDelta123) [reply src/wgextender/features/flags/FlagRegistration.java]: OK, I understood

Comment (Shevchik) [reply src/wgextender/features/flags/FlagRegistration.java]: Then remove the comment and bring back suppress warnings so it no longer shows as a diff.

**Comment by FrostDelta123** [comment]
Made

**Comment by RainbowKiwiFOX** [comment]
[18:09:16 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'rg claim testregion' in wgextender.features.claimcommand.WGRegionCommandWrapper(region)
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:161) ~[patched_1.13.1.jar:git-Paper-369]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:729) ~[patched_1.13.1.jar:git-Paper-369]
        at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1743) ~[patched_1.13.1.jar:git-Paper-369]
        at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1547) ~[patched_1.13.1.jar:git-Paper-369]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[patched_1.13.1.jar:git-Paper-369]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.13.1.jar:git-Paper-369]
        at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.1.jar:git-Paper-369]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_121]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_121]
        at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.1.jar:git-Paper-369]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.1.jar:git-Paper-369]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:434) ~[patched_1.13.1.jar:git-Paper-369]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.1.jar:git-Paper-369]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.1.jar:git-Paper-369]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
Caused by: java.lang.NullPointerException
        at wgextender.features.claimcommand.BlockLimits.processClaimInfo(BlockLimits.java:57) ~[?:?]
        at wgextender.features.claimcommand.WGRegionCommandWrapper.execute(WGRegionCommandWrapper.java:67) ~[?:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:151) ~[patched_1.13.1.jar:git-Paper-369]
        ... 14 more

**Comment by Shevchik** [comment]
The links in the gradle build file needs to be updated too.

**Comment by FrostDelta123** [comment]
> org.bukkit.command.CommandException: Unhandled exception executing 'rg claim testregion' in wgextender.features.claimcommand.WGRegionCommandWrapper(region)
> at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:161) ~[patched_1.13.1.jar:git-Paper-369]
> at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:729) ~[patched_1.13.1.jar:git-Paper-369]
> at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1743) ~[patched_1.13.1.jar:git-Paper-369]
> at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1547) ~[patched_1.13.1.jar:git-Paper-369]
> at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[patched_1.13.1.jar:git-Paper-369]
> at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.13.1.jar:git-Paper-369]
> at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12) ~[patched_1.13.1.jar:git-Paper-369]
> at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_121]
> at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_121]
> at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109) ~[patched_1.13.1.jar:git-Paper-369]
> at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016) ~[patched_1.13.1.jar:git-Paper-369]
> at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:434) ~[patched_1.13.1.jar:git-Paper-369]
> at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943) ~[patched_1.13.1.jar:git-Paper-369]
> at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841) ~[patched_1.13.1.jar:git-Paper-369]
> at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
> Caused by: java.lang.NullPointerException
> at wgextender.features.claimcommand.BlockLimits.processClaimInfo(BlockLimits.java:57) ~[?:?]
> at wgextender.features.claimcommand.WGRegionCommandWrapper.execute(WGRegionCommandWrapper.java:67) ~[?:?]
> at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:151) ~[patched_1.13.1.jar:git-Paper-369]
> ... 14 more

this is the reason why i changed Vault Integration. Runtime errors

**Pull request closed without merging**